modules/combined/test/tests/invOpt_mechanics/tests

```diff
@@ -0,0 +1,15 @@
+[Tests]
+  issues = '#21885'
+  design = 'OptimizationReporter.md'
```

Comment:
The hessian optimization isn't documented in an md file yet.  It will probably be documented in:
InvOptTheory.md
Can md in the optimization module reference input files from the combined module?  I wanted to document the combined module examples but I didn't know how.